assets/scss/base/form/_form.scss

```diff
@@ -0,0 +1,330 @@
+/*
+Group
+
+A Form field is used to group several controls or elements within a web form.
+
+Markup:
+<fieldset class="{{modifier_class}}">
+  <div class="form-field">
+    <label for="first-name">
+      <input type="text" id="first-name" value=""/>
+      First Name
+    </label>
+  </div>
+ <div class="form-field">
+    <label for="second-name">
+      <input type="text" id="second-name" value=""/>
+      Second Name
+    </label>
+  </div>
+  <div class="form-field">
+    <button class="button" type="submit">Send</button>
+  </div>
+</fieldset>
+
+.form-field-group    - A form field group can be applied via `.form-field-group`
+.form-field-single   - A form field single can be applied via `.form-field-single`
+
```

Comment:
Yup, same, if you have a look at the styles they are identical. This is also an issue I am raising